Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -385,6 +385,13 @@ final class PledgeViewController: UIViewController, MessageBannerViewControllerP
         }
       }
 
+    self.viewModel.outputs.showWebHelp
+      .observeForUI()
```

Comment:
Being the closest output to presenting the controller this one should be `observeForControllerAction()`